VotingApplication/VotingApplication.Web/Views/Routes/ManageInvitees.cshtml

```diff
@@ -1,5 +1,5 @@
 ﻿<div ng-app="GVA.Creation" ng-controller="ManageInviteesController">
-    <div class="centered">
+    <div class="manage-area centered">
```

Comment:
Can we bump up the width of the manage-area too? This currently reintroduces the issue where each email address almost always takes up a whole line